### DISCUSSION TITLE
Training model for road segmentation on my Duckietown dataset

### DISCUSSION THREAD

**Discussion by hellianet** [qubvel/segmentation_models.pytorch#570] [Q&A]
Hello! All my questions will be on the code that is presented at this link : https://github.com/qubvel/segmentation_models.pytorch/blob/master/examples/cars%20segmentation%20(camvid).ipynb
 1) I would like to know why the segmentation masks is needed during the test of the trained model? Shouldn't the trained model only accept pictures as input? How does it then differ from the untrained model?
 2) As a template, I used the code provided at the link above. I needed to train a DeepLabv3 neural network on my Duckietown simulator dataset. As a result, I got a very strange predicted mask. I don't understand why it happened... Please, tell me what could have affected this mask so much?
![Снимок экрана 2022-02-25 203706](https://user-images.githubusercontent.com/54978546/155731256-63924d8c-9e74-4366-bf0c-c63553a87bde.png)
![Снимок экрана 2022-02-25 203817](https://user-images.githubusercontent.com/54978546/155731266-4811e365-7f49-496a-9107-1e30a1777489.png)
